src/main/java/com/mineboundteam/minebound/item/registry/ItemRegistry.java

```diff
@@ -133,4 +133,21 @@ public class ItemRegistry {
     /* Light */
 
     /* Necrotic */
+
+    /* Dummy Spells */
+    public static final RegistryObject<Item> GUI_TELEKINETIC_OFFENSIVE_1 = ITEMS.register("magic/magic_telekinetic_o1", () -> new TelekineticOffensiveSpell(new Item.Properties().stacksTo(1), MagicConfigRegistry.TELEKINETIC_OFFENSIVE_1));
```

Comment:
Is it possible to, instead of getting a new one, just call on the Registry's existing entry? That way we won't be duplicating memory, it'll just duplicate the pointer